src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -222,6 +222,10 @@ typedef struct {
     struct MPIDI_OFI_win_request *deferredQ;
     MPIDI_OFI_win_targetinfo_t *winfo;
 
+    MPL_gavl_tree_t *dwin_target_mems;  /* MR keys registered to remote processes.
+                                         * One AVL tree per process. */
```

Comment:
I called it `target_mem` because the object contains both mr and addr.  See `MPIDI_OFI_target_mem_t` in ofi_types.h. But perhaps we can simply consider `target_mr` as an MPI-defined type which includes addr and OFI mr. Will change it.